### PR TITLE
Toast

### PR DESCRIPTION
**Pull request by JatuFaique**
Please Review Toast Component

### CODE REVIEW

**Comment by JatuFaique** [comment]
toast

File: components/toast/toast.css [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+  gap: 2px;
+}
+
+.disp-flex-row {
+  display: flex;
+  flex-direction: row;
+}
+
+.toast-bs {
+  max-width: 450px;
```
Comment (sonishreyas) [inline components/toast/toast.css:43]: you can change this to rem

**Review by sonishreyas — APPROVED** [review]
Just had one suggestion, Rest looks good to merge.

**Review by pratik-kumar-sahu — APPROVED** [review]
- get rid of commented code in final merge.
- Else looks good, go ahead with the merge.

**Pull request merged**